dynamic_vino_lib/src/inputs/standard_camera.cpp

```diff
@@ -18,15 +18,26 @@
  */
 #include "dynamic_vino_lib/inputs/standard_camera.hpp"
 
-// StandardCamera
 bool Input::StandardCamera::initialize()
 {
-  static int camera_count_ = 0;
-  setInitStatus(cap.open(camera_count_));
-  setWidth((size_t)cap.get(CV_CAP_PROP_FRAME_WIDTH));
-  setHeight((size_t)cap.get(CV_CAP_PROP_FRAME_HEIGHT));
-  camera_count_++;
-  return isInit();
+  static int camera_count_ = -1;
+  int fd; // A file descriptor to the video device
+  struct v4l2_capability cap;
```

Comment:
@luyang00 this part is not compatible with non-linux OS. and a MACRO to only enable it for Linux?

Reply:
thank you for your suggestion, v4l2 does only work for linux, I will add a MACRO later.

Reply:
Ok, I'll merge the code first, but I'll file a bug to record the fixing process in parallel.